packages/@ourworldindata/grapher/src/modal/SourcesModal.tsx

```diff
@@ -393,7 +394,7 @@ export class Source extends React.Component<{
             <div className="source">
                 {this.renderTitle()}
                 {this.def.descriptionShort && (
-                    <p>{this.def.descriptionShort}</p>
+                    <SimpleMarkdownText text={this.def.descriptionShort} />
```

Comment:
@sophiamersmann should we add the useParagraphs stanca here as well then?

Reply:
Since the old code wraps descriptionShort in a paragraph as well, I would leave it as is here.